firefox-ios/Shared/Prefs.swift

```diff
@@ -76,6 +76,7 @@ public struct PrefsKeys {
         public static let HistoryHighlightsSection = "HistoryHighlightsSectionUserPrefsKey"
         public static let InactiveTabs = "InactiveTabsUserPrefsKey"
         public static let JumpBackInSection = "JumpBackInSectionUserPrefsKey"
+        public static let CloseRemoteTabs = "CloseRemoteTabs"
```

Comment:
Is this still needed, now that you've switched to using the `FeatureFlagsBoolSetting` and `RustFxAFeatures`?

Reply:
Good catch! removed it and just directly used the `.closeRemoteTabs` in the Nimbus case statement.